api/src/test/java/org/openmrs/module/xdsbrepository/impl/XDSbServiceTest.java

```diff
@@ -811,16 +817,11 @@ public void provideAndRegisterDocumentSetB_shouldNotAddDuplcateIdsToAPatient() t
         //sanity checks that the patient already has the source patient id
         Patient patient = Context.getPatientService().getPatient(2);
         assertEquals(2, patient.getIdentifiers().size());
-        assertEquals("101-6", patient.getPatientIdentifier("1.2.3").getIdentifier());
+        assertEquals("1111111111", patient.getPatientIdentifier("1.2.3").getIdentifier());
```

Comment:
So it used to return a different id before? How many ids does this patient have?